packages/terra-form-select/src/native-select/NativeSelect.jsx

```diff
@@ -216,10 +216,10 @@ const NativeSelect = ({
     currentValue = getFirstValue(options, isFilterStyle);
     currentDisplay = getDisplay(currentValue, options, isFilterStyle, intl);
   }
-
+  const selectedText = intl.formatMessage({ id: 'Terra.form.select.selected' });
   const selectAttrs = {
     'aria-describedby': ariaDescribedBy,
-    'aria-label': ariaLabel,
+    'aria-label': (uncontrolledValue || value !== undefined) ? `${ariaLabel} ${selectedText}` : ariaLabel,
```

Comment:
(,) separator between arialabel and selected text will enhance the screen reader response. 
(value) is same as (value !== undefined) ??
```suggestion
    'aria-label': (uncontrolledValue || value ) ? `${ariaLabel}, ${selectedText}` : ariaLabel,
```

Reply:
Updated check condition and added separator - [865c9cc](https://github.com/cerner/terra-core/pull/3860/commits/865c9cc8226bedc26815dad21a509bbf06e15b34)